### PR TITLE
Rel Notes doc text for 4.6.2 release

### PR DESCRIPTION
**Pull request by jeana-redhat**

### CODE REVIEW

File: release_notes/ocp-4-6-release-notes.adoc [hunk 1]
```diff
@@ -2530,3 +2530,14 @@ An update for `jenkins-2-plugins`, `openshift-clients`, `podman`, `runc`, and `s
 Issued: 2020-10-27
 
 An update for several images is now available for {product-title} 4.6. Details of the update are documented in the link:https://access.redhat.com/errata/RHSA-2020:4298[RHSA-2020:4298] advisory.
+
+[id="ocp-4-6-2"]
+=== RHBA-2020:4339 - {product-title} 4.6.2 image release and bug fix advisory
+
+Issued: 2020-11-09
+
+{product-title} release 4.6.2 is now available. The list of container images and bug fixes includes in the update are documented in the link:https://access.redhat.com/errata/RHBA-2020:4339[RHBA-2020:4339] advisory. The RPM packages included in the update are provided by the link:https://access.redhat.com/errata/RHBA-2020:4340[RHBA-2020:4340] advisory.
```
Comment (bobfuru) [inline release_notes/ocp-4-6-release-notes.adoc:2539]: s/includes/included

**Review by bobfuru — COMMENTED** [review]
Left one comment and presuming all the links will work on 2020-11-09, and otherwise LGTM!

File: release_notes/ocp-4-6-release-notes.adoc (hunk 1 above)
Comment (jeana-redhat) [reply release_notes/ocp-4-6-release-notes.adoc:2539]: How did that even happen..? Thanks! :smiley:

**Comment by vikram-redhat** [comment]
Sorry there are 3 commits. I had to merge this quickly.

**Pull request merged**